packages/opds-web-client/src/app.tsx

```diff
@@ -66,6 +66,15 @@ class OPDSWebClient {
 
   render() {
     let RouteHandler = this.RouteHandler;
+
+    // `react-axe` should only run in development and testing mode.
+    // Running this is resource intensive and should only be used to test
+    // for accessibility and not during active development.
+    if (process.env.TEST_AXE === "true") {
```

Comment:
Does this part of the code not run in the browser?

Reply:
Oh I see, you define it in webpack

Reply:
`process.env` is a little confusing because that's normally used for environment variables on the server

Reply:
Yes, that is true but that's the convention that is usually used, even in files that will be used in the browser only.